### PR TITLE
[13.0] [ADD] product_pricelist: added operating_unit to model product_pricel…

### PR DESCRIPTION
**Pull request by jesus01x**
### Proposed changes

This module allow use operating unit in Product Pricelist.

### Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Migration Update

### Checklist

- [x] I have read the CONTRIBUTING doc
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated necessary documentation

### CODE REVIEW

**Comment by jesus01x** [comment]
@alan196

File: sale_operating_unit/README.rst [hunk 1]
```diff
@@ -13,21 +13,23 @@ Operating Unit in Sales
 .. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
     :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
     :alt: License: LGPL-3
-.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Foperating--unit-lightgray.png?logo=github
-    :target: https://github.com/OCA/operating-unit/tree/13.0/sale_operating_unit
-    :alt: OCA/operating-unit
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fserver--auth-lightgray.png?logo=github
+    :target: https://github.com/OCA/server-auth/tree/13.0/sale_operating_unit
```
Comment (alan196) [inline sale_operating_unit/README.rst]: Check the command that generates README is pointing to operating-unit repository

**Review by alan196 — CHANGES_REQUESTED** [review]

File: sale_operating_unit/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -1,3 +1,5 @@
 This module extends the Sales capabilities of Odoo and introduces the operating
 unit to the Sales Order. Security rules are defined to ensure that users can
```
Comment (alan196) [inline sale_operating_unit/readme/DESCRIPTION.rst:2]:
```suggestion
unit to the Sales Order and Pricelist. Security rules are defined to ensure that users can
```

File: sale_operating_unit/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -1,3 +1,5 @@
...
 only display the Sales Orders in which they are allowed access to.
+unit to the Product Pricelist. Security rules are defined to ensure that users can
```
Comment (alan196) [inline sale_operating_unit/readme/DESCRIPTION.rst]: remove this duplicated text

**Review by alan196 — CHANGES_REQUESTED** [review]

File: sale_operating_unit/models/product_pricelist.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
```
Comment (alan196) [inline sale_operating_unit/models/product_pricelist.py:3]: `# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).`

**Review by alan196 — CHANGES_REQUESTED** [review]

**Review by alan196 — APPROVED** [review]
Technical Review

**Pull request closed without merging**